addons/calendar/static/src/views/fields/many2many_attendee.js

```diff
@@ -47,6 +50,9 @@ export class Many2ManyAttendee extends Many2ManyTagsAvatarField {
             if (noEmail) {
                 tag.noEmail = true;
             }
+            if (unavailablePartnerIds.find((partner) => (tag.resId == partner.resId))) {
+                tag.unavailableIcon = true;
```

Comment:
Can we rename to '"isUnavailable" while we're at it?

The fact that it toggles an icon in the interface is a bit too specific.